README.md

```diff
@@ -21,19 +21,21 @@ import AWSLocationXCF
 You can create an AuthHelper and use it with the AWS SDK:
 
 ``` swift
-// Create an authentication helper instance using an Amazon Location API Key
-func exampleAPIKeyLogin() {
+// Create an authentication helper using credentials from Cognito
+func exampleCognitoLogin() {
     let authHelper = AuthHelper()
-    let locationCredentialsProvider = authHelper.authenticateWithAPIKey(apiKey: "My-Amazon-Location-API-Key", region: "us-east-1")
+    let locationCredentialsProvider = authHelper.authenticateWithCognitoUserPool(identityPoolId: "My-Cognito-Identity-Pool-Id", region: "us-east-1")
     let locationClient = authHelper.getLocationClient()
 }
 ```
 
 ``` swift
-// Create an authentication helper using credentials from Cognito
+// Create an authentication helper using credentials from any AWS-Swift-SDK Credentials Provider
```

Comment:
Maybe we can hyperlink https://github.com/awslabs/aws-crt-swift/blob/main/Source/AwsCommonRuntimeKit/auth/credentials/CredentialsProvider.swift

Reply:
Done